modules/core/src/main/scala/gem/config/SmartGcalConfig.scala

```diff
@@ -0,0 +1,6 @@
+package gem
+package config
+
+import gem.enum.SmartGcalType
+
+final case class SmartGcalConfig(smartGcalType: SmartGcalType)
```

Comment:
It seems a little strange to have a case class that wraps an enum. If there is no other structure associated with `SmartGcalConfig` could we just use `SmartGcalType` directly rather than wrapping it? If not it seems like this type might work better as an enumerated ADT and then only use `SmartGcalType` for serialization. I don't have a strong opinion. What do you think?